src/models/response.ts

```diff
@@ -0,0 +1,5 @@
+export interface ResponseObject {
```

Comment:
add jsdoc documentation above as well as documentation for what the fields will hold because this object will likely be used by other developers